app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -22,6 +22,7 @@ def new
       )
       result = form.submit(new_params)
       @platform_authenticator = form.platform_authenticator?
+      @mobile = mobile?
```

Comment:
Might be good to have some test coverage which asserts that the controller assigns this value, in `webauthn_setup_controller_spec.rb`

Reply:
what if we exposed `mobile?` as a helper method so it's accessible in the views?

```
helper :mobile?
```

Reply:
Changed to helper_method and added additional tests.